pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -520,13 +523,37 @@ func (r *Builder) mapDisks(vm *model.Workload, persistentVolumeClaims []core.Per
 		bus = VirtioBus
 	}
 
+	var bootOrder *uint
+	var imagePVC *core.PersistentVolumeClaim
 	for _, pvc := range persistentVolumeClaims {
+		// Handle loopvar https://go.dev/wiki/LoopvarExperiment
```

Comment:
it would be great to upgrade to go 1.21 when we can to get not only to get rid of this thingie but also to have slice.Contains ;)